script/config/ConfiguredScript.sol

```diff
@@ -42,13 +42,49 @@ abstract contract ConfiguredScript is Script {
         return vm.parseJson(vm.readFile(configPath));
     }
 
-    function _deployCreate2Code(string memory what, bytes memory args, bytes32 salt) internal returns (address addr) {
-        bytes memory bytecode = abi.encodePacked(vm.getCode(what), args);
+    function _deployCode(string memory submodule, string memory what, bytes memory args)
+        internal
+        returns (address addr)
+    {
+        vm.broadcast();
+        addr = deployCode(string.concat("lib/", submodule, "/out/", what, ".sol/", what, ".json"), args);
+
+        _logDeployment(submodule, what, args, addr);
+    }
+
+    function _deployCreate2Code(string memory submodule, string memory what, bytes memory args, bytes32 salt)
+        internal
+        returns (address addr)
+    {
+        bytes memory bytecode =
+            abi.encodePacked(vm.getCode(string.concat("lib/", submodule, "/out/", what, ".sol/", what, ".json")), args);
 
+        vm.broadcast();
         assembly ("memory-safe") {
             addr := create2(0, add(bytecode, 0x20), mload(bytecode), salt)
         }
 
         require(addr != address(0), "create2 deployment failed");
+
+        _logDeployment(submodule, what, args, addr);
+    }
+
+    function _logDeployment(string memory submodule, string memory what, bytes memory args, address addr)
+        internal
+        view
+    {
+        console2.log("Deployed %s at: %s", what, addr);
+
+        console2.log("");
+        console2.log("Verify %s using:", what);
+        console2.log("  > cd %s/", submodule);
+        console2.log(
```

Comment:
cool only to copy paste later on right?

Reply:
That's correct! In #15, I instead write these commands to a sh script so nobody has to copy/paste, it is automatically saved to the file system